tests/btrfs-progs/install.pm

```diff
@@ -11,6 +11,8 @@ use base 'opensusebasetest';
 use utils;
 use testapi;
 use serial_terminal 'select_serial_terminal';
+use version_utils qw(is_sle is_leap is_tumbleweed is_alp is_transactional);
```

Comment:
```suggestion
use version_utils 'is_transactional';
```
Looks only is_transactional used in install.pm, better remove others not in use.

Reply:
LGTM